assets/slackreport.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
nit: could you rename this as `.j2.json`, just to know that it is a template while browsing the  file tree? (not critical, feel free to disregard)